src/composables/use-media-query.js

```diff
@@ -41,8 +46,11 @@ export function useMediaQuery(query, options = { shouldPassInSSR: false }) {
 }
 
 /**
- * Check whether the curent screen meets
+ * Check whether the current screen meets
  * or exceeds the provided breakpoint size.
+ * @param {'sm'|'md'|'lg'|'xl'|'2xl'} breakpointName
+ * @param {Parameters<typeof useMediaQuery>} options
```

Comment:
Nice! Just one little fix. `Parameters` returns a tuple of the types for the function's arguments, and `options` is the second one argument so we need to pluck that one specifically.

```suggestion
 * @param {Parameters<typeof useMediaQuery>[1]} options
```

Now that you've added types to this file, if you feel up for it, it might be worth enabling TypeScript checking on this file by adding it to the `includes` list in `tsconfig.json` 😱